lib/skate/application.ex

```diff
@@ -103,4 +107,13 @@ defmodule Skate.Application do
   defp runtime_config(value) do
     value
   end
+
+  defp migrate() do
```

Comment:
@skyqrose outside the scope of this PR, but we should talk about what provisions we might want to make for being able to roll back migrations in prod. Maybe there's already some consensus on what a best practice here is.